lib/containers/common.pm

```diff
@@ -79,7 +79,7 @@ sub install_docker_when_needed {
                 # Temporarly enable LTSS product on LTSS systems where it is not present
                 my $ltss_needed = 0;
                 if (get_var('SCC_REGCODE_LTSS') && script_run('test -f /etc/products.d/SLES-LTSS.prod') != 0) {
-                    add_suseconnect_product('SLES-LTSS', undef, undef, '-r ' . get_var('SCC_REGCODE_LTSS'));
+                    add_suseconnect_product('SLES-LTSS', undef, undef, '-r ' . get_var('SCC_REGCODE_LTSS'), 150);
```

Comment:
@coolgw 

As I remembered, Oliver mentioned this function in the meeting "script_retry" for some network issue, is it suitable for you [case by case]?